modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -199,12 +219,13 @@ inline void preprocess(const cv::Mat& src,
                        const cv::gimpl::onnx::TensorInfo& ti,
                              cv::Mat& dst) {
     GAPI_Assert(src.depth() == CV_32F || src.depth() == CV_8U);
-
+    // CNN input type
+    const auto type = toCV(ti.type);
```

Comment:
BTW why it's become `type` if logically it's `depth`

Reply:
Fixed.

Reply:
Because it is `ONNX_TENSOR_ELEMENT_DATA_TYPE_`. From these thoughts.